src/main/resources/Untitled2.sql

```diff
@@ -0,0 +1,150 @@
+--Homework 07.02.2019:
+SELECT DISTINCT COMPANY FROM ORDERS o1 INNER JOIN PRODUCTS p1 ON o1.PRODUCT = p1.product_id INNER JOIN CUSTOMERS c1 ON o1.cust = c1.cust_num
```

Comment:
how would you select if you need all the information and not just a name of a company?

Reply:
It is ok. But if you use alias for table names use it for columns too. c1.COMPANY in your case